client/components/app-promo-card/stories/index.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import { AppPromoCard } from '..';
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta: Meta< typeof AppPromoCard > = {
+	title: 'client/components/App Promo Card',
+	component: AppPromoCard,
+};
+export default meta;
+
+type Story = StoryObj< typeof AppPromoCard >;
+
+export const Default: Story = {};
```

Comment:
Cleaning up the stories. The old stories contained unnecessary separate stories for Woo and Jetpack, and here, we have auto-generated controls where the `isWoo` prop can be toggled on or off.